### ISSUE TITLE
Bring back mascots

### ISSUE THREAD

**Issue by lc-guy** [nebukazar/StyleChan#1]
Why are they gone?

**Comment by aholfditler** [comment]
you are looking for oneechan by the same developer. https://github.com/nebukazar/OneeChan

**Comment by lc-guy** [comment]
nah, I know this is a new fork: I was just wondering why he was removing them as part of oneechan's "unused features".

**Comment by nebukazar** [comment]
I don't use mascots personally and they do make the script a bit sluggish when loading or saving options, you need to call cross site links from hosting the images etc.

Thus Stylechan.

**Comment by JoelNichols** [comment]
So is Stylechan essentially Oneechan minus the mascots? I tried it briefly and it did seem to have some interesting (new?) features but the lack of mascots made me sad.

**Comment by nebukazar** [comment]
Basically yes, nothing changes. I removed the mascots and a few settings like style thread stats, checkboxes, SS-like Sidebar and compatibility with 4chan X v2 (the old version).

Both versions will get fixes.

**Comment by JoelNichols** [comment]
Excellent, many thanks for clarifying that.
